### PR TITLE
[Fix] Membership requests are not being displayed on the Accept Membership Requests page

### LINKED ISSUE #563: [Bug] Membership requests are not being displayed on the Accept Membership Requests page
- **Issue Description:** Membership requests are not being displayed on the Accept Membership Requests page
- **Issue Severity:** High
- **Environment Details:** Issue found in `talawa\lib\views\pages\organization\accept_requests_page.dart`
- **Observed Behaviour:** No request is displayed even if they are present.
- **Expected Behaviour:** Membership requests should be displayed
- **Steps to reproduce issue:** 
   

> 1. create a new private organization with one ID.
> 2. now login with another ID and send a membership request to the above-created organization and after that log out from this ID.
> 3. now log in again with the ID which was used to create the private org and visit the  Accept membership requests  section of the Organization Settings  page. 
> 4. Wait for the requests to load.

- **Snapshots/Videos:**
  

>    parentData: <none> (can use size)
>     constraints: BoxConstraints(0.0<=w<=352.7, 0.0<=h<=Infinity)
>    size: MISSING
>     axisDirection: down
>     crossAxisDirection: right
 >    offset: ScrollPositionWithSingleContext#e9d0f(offset: 0.0, range: null..null, viewport: null, ScrollableState, AlwaysScrollableScrollPhysics -> ClampingScrollPhysics -> RangeMaintainingScrollPhysics, IdleScrollActivity#2922d, ScrollDirection.idle)
>     anchor: 0.0
>    center child: RenderSliverPadding#08024 NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
>         parentData: paintOffset=Offset(0.0, 0.0)
>         constraints: MISSING
>         geometry: null
>         padding: EdgeInsets.zero
>         textDirection: ltr
>         child: RenderSliverList#d7a24 NEEDS-LAYOUT NEEDS-PAINT
>             parentData: paintOffset=Offset(0.0, 0.0)
>            constraints: MISSING
>             geometry: null
>             no children current live
> > ════════════════════════════════════════════════════════════════════════════════
> 
> ════════ Exception caught by rendering library ═════════════════════════════════
> RenderBox was not laid out: RenderViewport#1d699 NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
> 'package:flutter/src/rendering/box.dart':
> Failed assertion: line 1940 pos 12: 'hasSize'

> The relevant error-causing widget was
> ListView
> lib\…\organization\accept_requests_page.dart:195
> >════════════════════════════════════════════════════════════════════════════════
> 
> 
- **Does this issue need immediate attention?** Yes, because without resolving this issue private organizations will not be able to accept membership requests.
- **Are you willing to work on this issue:** Yes, I would love to work on this issue. Please assign this to me!

### PR DESCRIPTION
**Pull request by Raunakk02**
**What kind of change does this PR introduce?**
[bugfix] Membership requests are now being displayed on the Accept Membership Requests page which was not displayed previously.

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
No

**Summary**
Resolves #563 

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by Sagar2366** [comment]
@Raunakk02 add problem and solution video.
Also, CI is failing please check.

**Review by Sagar2366 — CHANGES_REQUESTED** [review]
> @Sagar2366 I have merged the recent changes from upstream/master.

add problem and solution video.
Disable code-autoformatter so that only changes you added will get added to commit. Once done, send PR again for review.

File: lib/views/pages/organization/accept_requests_page.dart [hunk 1]
```diff
@@ -70,10 +69,9 @@ class _AcceptRequestsPageState extends State<AcceptRequestsPage> {
     //this function give the functionality of accepting the request of the user by the administrator
     GraphQLClient _client = graphQLConfiguration.authClient();
 
-    QueryResult result = await _client.query(QueryOptions(
-        documentNode: gql(_query.acceptMembershipRequest(itemIndex))));
-    if (result.hasException &&
-        result.exception.toString().substring(16) == accessTokenException) {
+    QueryResult result = await _client
```
[the remark this answers is not in the document]
Comment (Raunakk02) [reply lib/views/pages/organization/accept_requests_page.dart:72]: Sure, working on it!

File: lib/views/pages/organization/accept_requests_page.dart [hunk 2]
```diff
@@ -98,10 +96,9 @@ class _AcceptRequestsPageState extends State<AcceptRequestsPage> {
     //this function give the functionality of rejecting the request of the user by the administrator
     GraphQLClient _client = graphQLConfiguration.authClient();
 
-    QueryResult result = await _client.query(QueryOptions(
-        documentNode: gql(_query.rejectMembershipRequest(itemIndex))));
-    if (result.hasException &&
-        result.exception.toString().substring(16) == accessTokenException) {
+    QueryResult result = await _client
```
[the remark this answers is not in the document]
Comment (Raunakk02) [reply lib/views/pages/organization/accept_requests_page.dart:99]: Sure, working on it!

**Comment by Raunakk02** [comment]
@Sagar2366 I have changed the code which you marked back to the original format and also disabled the code auto-formatter before doing so. 
Awaiting review!

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**